examples/README.md

```diff
@@ -3,6 +3,7 @@
 The repository includes the following example scripts:
```

Comment:
I wonder if it will be better to have two separate folders, one for single-molecule property prediction, one for pdbbind.

Reply:
Sounds good - I will reorganize into folders